doc/api/console.md

```diff
@@ -417,11 +417,114 @@ added: v0.1.100
 
 The `console.warn()` function is an alias for [`console.error()`][].
 
+## Inspector only methods
+The following methods are exposed by the V8 engine in the general API but do
+not display anything unless used in conjunction with the inspector tab
```

Comment:
having `inspector` be a link to `debugger.html` and/or `inspector.html` would be good here. `debugger.html` is the best choice.